PSOperations/LocationOperation.swift

```diff
@@ -30,7 +30,11 @@ public class LocationOperation: Operation, CLLocationManagerDelegate {
         self.accuracy = accuracy
         self.handler = locationHandler
         super.init()
-        addCondition(LocationCondition(usage: .WhenInUse))
+        #if !os(tvOS)
+            addCondition(Capability(Location.WhenInUse))
```

Comment:
Could be nice to specify permissions level (`WhenInUse` vs `Always`) as an optional parameter that defaults to `WhenInUse` in this `init`. That could be left for a later PR though as you're not actually changing the behavior here.